roles/realm_join/tasks/install_pexpect.yml

```diff
@@ -0,0 +1,37 @@
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: python3-pexpect
+    state: latest
+  # on Fedora this will be True due to the major version. Not nice, but works.
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version is version('8', '>')
+
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: python-pexpect
+    state: latest
+  # on Fedora this will be True due to the major version. Not nice, but works.
```

Comment:
I hightly doubt that `ansible_os_family: Debian` will ever be true on Fedora ;)

I think this comment can co now.

Reply:
xD yes, indeed ....